src/libsyntax_pos/symbol.rs

```diff
@@ -942,6 +987,15 @@ pub mod kw {
 pub mod sym {
     use super::Symbol;
     symbols!();
+
+    // Get the symbol for an integer. Integers 0-9 have a static symbol and
```

Comment:
```suggestion
    // Get the symbol for an integer. A few small integers have static symbols and
```
(Users of the function don't necessarily know the specific details.)